layers/layers.config.yaml

```diff
@@ -0,0 +1,12 @@
+indexMappings:
+  https://lschema.org/attributeName: 
```

Comment:
These are objects, not elements of an array

  - https://lschema.org/attributeName
etc.